CMSgov/mint-app
# [EASI-4443] Data Exchange Concept Help Article

# EASI-4443


## Changes and Description

- Add Data Exchange Concept help article
- Add unit test



## How to test this change

1. Go to `http://localhost:3005/help-and-knowledge`
2. Find new article listed there
3. Click `Browse all help articles` and verify that the new `Evaluating data exchange concepts` article is first
4. Click `Read` 
5. Verify that the help page matches figma and meets AC



## PR Author Review Checklist



## PR Reviewer Guidelines
- It's best to pull the branch locally and test it, rather than just looking at the code online!
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
- Always make comments, even if it's minor, or if you don't understand why something was done.
